stubs/pywin32/pythoncom.pyi

```diff
@@ -63,7 +63,7 @@ def GetRunningObjectTable(reserved: int = ...) -> _win32typing.PyIRunningObjectT
 def GetScodeString(scode) -> str: ...
 def GetScodeRangeString(scode) -> str: ...
 def GetSeverityString(scode) -> str: ...
-def IsGatewayRegistered(iid: _win32typing.PyIID) -> bool: ...
+def IsGatewayRegistered(_iid: _win32typing.PyIID | None) -> int: ...
```

Comment:
Did you mean to make the param positional-only? That needs two underscores not one.

Reply:
I did! Fixed.
I also think all of the C-extension modules don't support keyword-arguments. But it would be too much of a chore to validate them all.

Reply:
Thanks!

> all methods in C-extension modules don't support keyword-arguments 

Might be true for pywin32, but it's possible to support kwargs by using `PyArg_ParseTupleAndKeywords`.